src/test/java/SetCollectionTest.java

```diff
@@ -39,7 +39,6 @@ void checkHasValueOnlyTrue(int input) {
     @ParameterizedTest
     @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
     void checkHasValue(String input, String expected) {
```

Comment:
```suggestion
    void checkHasValue(int input, boolean expected) {
```

이 부분은 포비님도 말씀주셨듯 자동 형변환되어서 수정해주셔도 좋을 것 같습니다 : )

Reply:
반영했습니다~!